tests/unit_tests/test_wave_06.py

```diff
@@ -143,7 +161,21 @@ def test_swap_best_by_category_reordered():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    # Assert
+    assert result == True
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert item_a in tai.inventory
+    assert item_b in tai.inventory
+    assert item_f in tai.inventory
+    assert item_c not in tai.inventory
```

Comment:
Your asserts thoroughly check the list contents by checking their lengths, and each of the 3 items we would expect to find. If a list has three items, and we find the three we expect, then logically there can be no other items in that list. As a result, it's not necessary to check that the swapped items are not in their original lists. Since we've checked every item, and none of them are the item that was swapped out, we know the swapped out item is not in the list.